06-release-engineering-analytics.Rmd

```diff
@@ -147,8 +149,8 @@ publications were stored centrally, accessible to all reviewers.
 
 ### Study Selection
 
-In the utilized tool for conducting the survey, it is possible to label papers
-with tags and leave comments and ratings.
+In the aforementioned tool for storing the papers,
```

Comment:
Maybe even explicitly state that we used the tool for selection purposes? Something along the lines of:
"To select papers for inclusion in the survey, we used the aforementioned tool..."